docs/includes/topic_f5lbaas-l7_content_switching.rst

```diff
@@ -0,0 +1,197 @@
+            +----------------------+-------------------------+------------------------------------------+
+            | ENDS_WITH            | No                      | Ends with                                |
+            +----------------------+-------------------------+------------------------------------------+
+            | ENDS_WITH            | Yes                     | Does not end with                        |
+            +----------------------+-------------------------+------------------------------------------+
+            | EQUAL_TO             | No                      | Is                                       |
+            +----------------------+-------------------------+------------------------------------------+
+            | EQUAL_TO             | Yes                     | Is not                                   |
+            +----------------------+-------------------------+------------------------------------------+
+            | CONTAINS             | No                      | Contains                                 |
+            +----------------------+-------------------------+------------------------------------------+
+            | CONTAINS             | Yes                     | Does not contain                         |
+            +----------------------+-------------------------+------------------------------------------+
+            | REGEX                | X                       | No direct mapping                        |
+            +----------------------+-------------------------+------------------------------------------+
+
+        All L7Policy actions map directly to TMOS traffic policy rule actions:
+
+            +------------------+---------------------------------+
+            | L7 Policy Action | TMOS Traffic Policy Rule Action |
+            +==================+=================================+
+            | Reject           | Reset traffic                   |
+            +------------------+---------------------------------+
+            | RedirectToUrl    | Redirect                        |
+            +------------------+---------------------------------+
+            | RedirectToPool   | Forward traffic to pool         |
+            +------------------+---------------------------------+
+
+Since the L7 Rules are expected to match on each HTTP request, if a listener has an L7Policy attached, the presence of a OneConnect profile on the corresponding TMOS virtual server must be assured.
+
+Initially, the support for Regex is not available.  Since there is a direct mapping of the other compare types in Neutron to BIG-IP LTM, and regular expressions can in some cases be expressed as the logical ‘OR’ of multiple string compares, we propose an initial implementation of L7 Policies and Rules without regular expressions with the understanding that it will be available as a feature in a future release.
```

Comment:
Good to know internally, but doesn't really need to be in the public documentation. It's enough to say in the caveats that this release doesn't include support for regex.